VideoLocker/res/layout/edx_spinner_item.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@android:id/text1"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:ellipsize="marquee"
+    android:paddingBottom="@dimen/vertical_input_padding"
+    android:paddingLeft="@dimen/horizontal_input_padding"
+    android:paddingRight="@dimen/horizontal_input_padding"
+    android:paddingTop="@dimen/vertical_input_padding"
+    android:singleLine="true"
+    android:textAlignment="inherit"
```

Comment:
Copied from the `android.R.layout.simple_spinner_item` but you're right we don't need it.